readme.md

```diff
@@ -2,29 +2,24 @@
 
 > Get user info from a GitHub username
 
-
 ## Install
 
 ```
 $ npm install gh-user
 ```
 
-
 ## Usage
 
 ```js
 const ghUser = require('gh-user');
 
-(async () => {
-	console.log(await ghUser('sindresorhus'));
-	//=> {login: 'sindresorhus', id: 170270, …}
-})();
+console.log(await ghUser('sindresorhus'));
+//=> {login: 'sindresorhus', id: 170270, …}
 ```
 
-
 ## API
 
-### ghUser(username, [options])
+### ghUser(username, token?)
```

Comment:
I think it should stay with an options object even if it's just one option. More readable and makes it easier to add more options in the future.